src/translations/translations.rs

```diff
@@ -445,6 +460,7 @@ pub fn network_adapter_translation(language: Language) -> &'static str {
         Language::FI => "Verkkosovitin",
         Language::JA => "ネットワーク アダプター",
         Language::UZ => "Tarmoq adapteri",
+        Language::VN => "Network adapter",
```

Comment:
missing translation?

Reply:
I think that keep it like that is fine, translate it into Vietnamese is kinda long and might be misunderstood, confusing.  We  call it Network adapter directly most of the times. :)